### ISSUE TITLE
Server-side support

### ISSUE THREAD

**Issue by hacen-ai** [AndreSouzaAbreu/vue-data-table#29]
Hello, does is have server side support?

Will be easy to adapte it to  https://github.com/izimobil/django-rest-framework-datatables ?

HAve a nice day

**Comment by AndreSouzaAbreu** [comment]
No, it doesn't have server side rendering support, but I think it is a nice
feature to have and I'd be willing to implement it or to accept pull requests.

However, I don't know what should be done for that matter. Do you have any idea
about it? I have used Vue only for SPA, not for SSR.

**Comment by uwla** [comment]
I have built a nuxt app using this plugin and SSR is working to render it.
Nuxt has a set of tools to provide SSR.

Does that work for you?

**Comment by uwla** [comment]
@hacen-ai can you give me an update?
